scene/resources/visual_shader.h

```diff
@@ -639,6 +639,7 @@ class VisualShaderNodeParameterRef : public VisualShaderNode {
 	virtual PortType get_output_port_type(int p_port) const override;
 	virtual String get_output_port_name(int p_port) const override;
 
+	bool get_shader_valid() const;
```

Comment:
```suggestion
	bool is_shader_valid() const;
```
I'd use this naming, `get_shader_valid` sounds like `return shader.is_valid() ? shader : RID()` to me

Reply:
Thanks!